config/certmanager/certificate-metrics.yaml

```diff
@@ -0,0 +1,17 @@
+# The following manifests contain a metrics certificate CR.
+# More document can be found at https://docs.cert-manager.io
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: metrics-cert  # this name should match the one appeared in kustomizeconfig.yaml
```

Comment:
helm review applies here as well

Reply:
I'll check any missing piece in helm configurations. But I tested kustomize configurations extensively and it works.

Reply:
I meant mostly this https://github.com/kubernetes-sigs/lws/pull/478/files#r2032107950